src/main/java/org/xbill/DNS/SimpleResolver.java

```diff
@@ -44,6 +44,8 @@ public class SimpleResolver implements Resolver {
 
   private static final short DEFAULT_UDPSIZE = 512;
 
+  private ResolverClientFactory clientFactory = new DefaultResolverClientFactory();
```

Comment:
What's the intended way to change this from the default to something else?

Reply:
I forgot the setter.